.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -24,6 +24,14 @@ body:
       description: Include MongoDB queries if you can.
     validations:
       required: true
+  - type: textarea
+    id: how
+    attributes:
+      label: How can we reproduce it?
+      description: >
+        Include a pull request with test script noted in [reporting a bug](../../CONTRIBUTING.md#reporting-a-bug) if you can.
+    validations:
+      required: true
```

Comment:
As the previous sentence says, "if you can", maybe this field shouldn't be required?

Reply:
good point. Let's make it not required!